test/ExpressiveInstallerTest/SetupDataAndCacheDirTest.php

```diff
@@ -17,17 +17,17 @@ class SetupDataAndCacheDirTest extends OptionalPackagesTestCase
     /**
      * @var OptionalPackages
      */
-    protected $installer;
+    private $installer;
```

Comment:
Why are you changing all of these to `private`? Is there a compelling reason? (I'm not convinced visibility matters much in test cases, to be honest.)